src/crimson/os/seastore/onode_manager/staged-fltree/node_extent_manager/seastore.h

```diff
@@ -106,16 +106,19 @@ class SeastoreNodeExtentManager final: public TransactionManagerHandle {
       }
     }
     return tm.read_extent<SeastoreNodeExtent>(t, addr
-    ).safe_then([addr, &t](auto&& e) {
+    ).safe_then([addr, &t](auto&& e) 
+      -> read_ertr::future<NodeExtentRef> {
       TRACET("read {}B at {:#x} -- {}",
              t, e->get_length(), e->get_laddr(), *e);
+      //XXX: this should be remove when interruptible futures are involved
       if (!e->is_valid()) {
         ERRORT("read invalid extent: {}", t, *e);
```

Comment:
Not an error any more, maybe `INFOT()` instead?

Reply:
same here. no need check extent valid.

Reply:
> Not an error any more, maybe `INFOT()` instead?

ok

Reply:
> same here. no need check extent valid.

I think this is the same case as [here](https://github.com/ceph/ceph/pull/41867/files#r653158689)